src/staff_ghosts.c

```diff
@@ -507,34 +521,36 @@ void func_80005B18(void) {
                 D_80162DD0 = D_80162DCC;
                 D_80162DE8 = gPlayerOne->characterId;
                 D_80162DD8 = 0;
-                D_80162DD4 = 0;
+                gPlayerGhostDisabled = 0;
                 D_80162DDC = 1;
                 func_80005AE8(gPlayerTwo);
                 func_80005AE8(gPlayerThree);
             }
-        } else {
-            if ((gLapCountByPlayerId[0] == 3) && (D_80162DDC == 0) && (D_80162DF8 == 1)) {
+        } 
+        else {
+            if ((gLapCountByPlayerId[0] == 3) && (D_80162DDC == 0) && (gReplayCannotSave == 1)) {
                 D_80162D80 = D_802BFB80.arraySize8[0][D_80162DC8][3].pixel_index_array;
                 D_80162D84 = D_80162D86;
                 D_80162DDC = 1;
             }
             if ((gPlayerOne->type & 0x800) == 0x800) {
                 func_80005AE8(gPlayerTwo);
                 func_80005AE8(gPlayerThree);
-            } else {
-                D_80162DEC += 1;
-                if (D_80162DEC >= 0x65) {
-                    D_80162DEC = 0x00000064;
+            }
+            else {
+                UNUSED_VAR2 += 1;
+                if (UNUSED_VAR2 >= 101) {
+                    UNUSED_VAR2 = 100;
```

Comment:
Comment bubble above was glitchy for me. In-case it didn't save. I was asking for var `> 100` instead of `>= 101` as that should match there and looks better.

Reply:
Done